systemtest/src/main/java/io/strimzi/systemtest/MessagingBaseST.java

```diff
@@ -79,6 +79,45 @@ void availabilityTest(int messageCount, long timeout, String clusterName, boolea
         assertSentAndReceivedMessages(sent, received);
     }
 
+    /**
+     * Method for send messages to specific kafka cluster. It uses test-client API for communication with deployed clients inside kubernetes cluster
+     * @param messageCount messages count
+     * @param timeout timeout for producer to be finished
+     * @param clusterName cluster name
+     * @param tlsListener option for tls listener inside kafka cluster
+     * @param topicName topic name
+     * @return count of send and acknowledged messages
+     */
+    int sendMessages(int messageCount, long timeout, String clusterName, boolean tlsListener, String topicName) throws Exception {
```

Comment:
Out fo curiosity . don't we have some methods for sending and receiving messages elsewhere? It seems to me like we have to use this already somewhere in other tests.

Reply:
Correct, it was just overloaded method without using KafkaUser parameter.